daml-foundations/daml-ghc/tests/GenTemplCompat.daml

```diff
@@ -0,0 +1,66 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+-- Check that non-generic templates work with the new de/re-sugaring of
+-- templates for DAML-LF < 1.5 as well. This test can be deleted when
```

Comment:
In your PR description you say that we need this change since we do not want to drop support for DAML-LF < 1.5 when generic templates land but here you say that we can delete the test when generic templates land. Is the reason for that that we will exercise the same codepath via another test or am I missing something here?

Reply:
Yes, when generic templates land, all other tests will use the code path currently only tested by this test.